services/web/server/src/simcore_service_webserver/activity/__init__.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+import logging
+
+from aiohttp import web
+from servicelib.application_keys import APP_CONFIG_KEY
+from servicelib.rest_routing import (get_handlers_from_namespace,
+                                     iter_path_operations,
+                                     map_handlers_with_operations)
+
+from ..rest_config import APP_OPENAPI_SPECS_KEY
+from . import handlers
+from .config import CONFIG_SECTION_NAME
+
+logger = logging.getLogger(__name__)
+
+def setup(app: web.Application,* , disable_login=False):
```

Comment:
with latest version in master, you will have to add the decorator here and then can remove some of the code below